deltachat-ffi/deltachat.h

```diff
@@ -3114,6 +3115,35 @@ int dc_receive_backup (dc_context_t* context, const char* qr);
  */
 dc_accounts_t* dc_accounts_new                  (const char* dir, int writable);
 
+/**
+ * Create a new account manager with an existing events channel,
+ * which allows you see events emitted during startup.
+ * 
+ * The account manager takes an directory
```

Comment:
```suggestion
 * The account manager takes a directory
```
Maybe it's better not to duplicate the documentation, but just add a reference to `dc_account_new()` and explain the difference. Otherwise API users need to read both and build a diff in their head, and we have more text to maintain.